content/ja/docs/tasks/debug/debug-application/determine-reason-pod-failure.md

```diff
@@ -90,11 +90,12 @@ spec:
 
 ## {{% heading "whatsnext" %}}
 
-
 * [コンテナ](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#container-v1-core)の`terminationMessagePath`フィールド参照
-* [ログ取得](/docs/concepts/cluster-administration/logging/)について
+* [イメージ](/ja/docs/concepts/containers/images/)の[ImagePullBackOff](/ja/docs/concepts/containers/images/#imagepullbackoff)参照
```

Comment:
```suggestion
* [イメージ](/ja/docs/concepts/containers/images/)の[ImagePullBackOff](/ja/docs/concepts/containers/images/#imagepullbackoff)を参照
```
This expression is more natural as  Japanese.

Reply:
Fixed, thanks!